Userland/Libraries/LibC/string.cpp

```diff
@@ -164,6 +168,8 @@ void* memset(void* dest_ptr, int c, size_t n)
         : "memory");
     return dest_ptr;
 }
+#elif !ARCH(X86_64)
```

Comment:
nit: Please make this a normal if/else check and then move the comment about the x86_64 assembler implementation into the `ARCH(X86_64)` check, and add a standard `#else` + Unknown architecture message.